### ISSUE TITLE
[BUG] The new image does not start

### ISSUE THREAD

**Issue by hufhend** [linuxserver/docker-unifi-controller#178]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

After updating the image to the new version 7.3.76 UniFi controller does not start

### Steps To Reproduce

cat docker-compose.yml 
```
version: "3.7"
services:
  unifi-controller:
    image: lscr.io/linuxserver/unifi-controller
    container_name: unifi-controller
    environment:
      - PUID=1000
      - PGID=1000
      - MEM_LIMIT=1024 #optional
      - MEM_STARTUP=1024 #optional    
    volumes:
      - "$PWD/data:/config"
    ports:
      - 3478:3478/udp
      - 10001:10001/udp
      - 8080:8080
      - 8443:8443
      - 1900:1900/udp #optional
      - 8843:8843 #optional
      - 8880:8880 #optional
      - 6789:6789 #optional
      - 5514:5514/udp #optional
    restart: always
```

### Environment

```markdown
- OS: Ubuntu 22.04.1 LTS, kernel 5.16.17-sun50iw9 aarch64
- How docker service was installed: Docker version 20.10.18, build b40c2f6
```

### CPU architecture

arm64

### Docker creation

```bash
lscr.io/linuxserver/unifi-controller
```

### Container logs

```
[...]
s6-rc: info: service 99-ci-service-check: starting
[ls.io-init] done.
s6-rc: info: service 99-ci-service-check successfully started
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.apache.catalina.loader.WebappClassLoaderBase (file:/usr/lib/unifi/lib/tomcat-embed-core-9.0.59.jar) to field java.io.ObjectStreamClass$Caches.localDescs
WARNING: Please consider reporting this to the maintainers of org.apache.catalina.loader.WebappClassLoaderBase
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release``

**Comment by thespad** [comment]
Those warning messages are expected since their bump to requiring JRE 11 (they need to tidy up their code).

You would need to check the application/db logs in the /config mount if the services don't appear to be starting.

**Comment by LowieHuyghe** [comment]
I'm also having issues with the new version of the image. Mine shows the following output in the container-logs:

```
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
Invalid maximum heap size: -Xmx512MM
```

**Comment by gsaurer** [comment]
For me both last versions 7.3.76 and 7.2.95 are not starting. Had to go back to 7.2.94.

**Comment by djlucas** [comment]
Seeing the same issue. Last working image is 7.2.94-ls165. For latest, nothing appears in the logs dir. I jumped into it, but I'm unfamiliar with s6 as of yet. What can we do to assist?

**Comment by mezzed-dev** [comment]
I'm seeing the same issue also. In my case I was able to roll back to an older version of the 7.2.95 image that was still on my system, but if I pull the new version of the 7.2.95 image or the 7.3.96 image, they will not load.

**Comment by vk1kcm** [comment]
I found if you have

` - MEM_LIMIT=1024M #optional`

in docker-compose then it will fail with a java error.
```
Error: A fatal exception has occurred. Program will exit.
Invalid maximum heap size: -Xmx1024MM

```

Fix is to change `1024M` to `1024`.

**Comment by mezzed-dev** [comment]
That worked for me! Thanks!!

**Comment by CyberFlexie** [comment]
Had the same problem. 

Changed from latest to: `image: lscr.io/linuxserver/unifi-controller:7.2.95` and problems went away...

**Comment by j0nnymoe** [comment]
Well if it was the same proiblem, the fix is 2 comments above yours.

Closing issue as fixed was posted.

**Comment by janusn** [comment]
I have the same issue. I have not set MEM_LIMIT in the docker compose yaml.

The problem went away only if I have down graded to /linuxserver/unifi-controller:-ls165.

@j0nnymoe would you please reopen this issue? It doesn't seem to fix for @djlucas nor @gsaurer either.

**Comment by Nulldev128** [comment]
> 

I could not get past those 'illegal reflective...' warnings listed above & get the application to start until I pulled 7.2.95 instead of latest. Thanks for the temporary fix, but will this issue be investigated so we can pull latest? Happy to help with that.

**Comment by TheColetrain** [comment]
I am certainly not the most tech savvy person. But I recently lost access to the Unifi controller running docker container, pretty much all that was said above.  I tried a few things and did a few googles:

-  tried reloading the stack
-  reloading with :latest
- remapping the volume
- deleting the container image and reloading


and it did not resolve the **`404 error`**  when attempting to access the web UI.  Couldn't get it to work until i did a complete wipe and reinstall.

I ended up deleting the stack, volume, container, image and reinstalling and restoring settings from backup and it appears to be working at this time.  however, the error code in the first part of this thread was my suspect, and even after reinstall still get that code in the logs.  I am not sure why it broke, I had not recently updated.

I am using portainer stack on ubuntu VM on proxmox.



I know that my info / data provided here is vague, but wanted to do a +1 to confirm something is going on.

thank you for all your work.
